src/node/glossary.ts

```diff
@@ -0,0 +1,5 @@
+import { OnUnhandledRequest } from '../onUnhandledRequest'
+
+export interface ListenOptions {
+  onUnhandledRequest?: OnUnhandledRequest
```

Comment:
Considering we will be shipping the same behavior in terms of this option to both Browser and Node, do you find it useful to introduce a concept of "shared" configuration?

```ts
interface SharedOptions {
  onUnhandledRequest?: OnUnhandledRequest
}

type WorkerOptions = SharedOptions & {
  serviceWorker: WhatWeHaveNow
  // ...
}

type ServerOptions = SharedOptions
```

All those interfaces, apart from the `SharedOptions`, already exist in the respective `setupWorker`/`setupServer` declarations.

Reply:
Perhaps we could call it `IsomorphicOptions`? Doesn't really matter to me.

Reply:
"Shared" seems to describe that the options are shared between multiple units. I'm not sure what "isomorphic" means in relation to those settings.